src/main/java/org/folio/services/financedata/FinanceDataValidator.java

```diff
@@ -115,15 +114,15 @@ private void validateRequiredField(List<Error> combinedErrors, String fieldName,
     }
   }
 
-  public Future<Void> validateIds(FyFinanceDataCollection financeDataCollection, RequestContext requestContext) {
+  public Future<Void> comparingWithExistingData(FyFinanceDataCollection financeDataCollection, RequestContext requestContext) {
```

Comment:
we can rename to compareWithExistingData to follow standard convention

Reply:
changed